### DISCUSSION TITLE
Email document from document page

### DISCUSSION THREAD

**Discussion by user34756361233** [jonaswinkler/paperless-ng#1579] [Feature Requests]
I find my self regularly in the situation I have to share a document with myself or somebody else.
Would it be possible to extend the pull-down menu for download with an action to attach the document to an email. A kind of send to option that kicks of the email client and attaches the document.
If I missed something in the manual and it is already possible... we'll I could not find it.

Great program and keep up the good work!

tia 
Han

**Comment by nilsherb** [comment]
Yes please .... :-)
